oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -137,6 +137,10 @@ core:
     trainingPeriodHttpUriRecognitionPattern: ${SW_CORE_TRAINING_PERIOD_HTTP_URI_RECOGNITION_PATTERN:60}
     # The max number of HTTP URIs per service for further URI pattern recognition.
     maxHttpUrisNumberPerService: ${SW_CORE_MAX_HTTP_URIS_NUMBER_PER_SVR:3000}
+    # Include the list of keywords to filter configurations including secrets. Separate keywords by a comma.
+    keywords4MaskingSecretsOfConfig: ${SW_CORE_KEYWORDS_FOR_MASKING_SECRETS:user,password,token,accessKey,secretKey}
+    # Disable core HTTP:port/debugging/config/dump
+    disableConfigDump: ${SW_DISABLE_CONFIG_DUMP:false}
```

Comment:
These are all about the new module, why should we add them in the core section?

Reply:
Because ServerStatusService is a core module service.

Reply:
> I am thinking of the other features, such as L1/L2 queue status dump, and cache dump.

I believe those should be done through self-observability metrics